unify_idents/engine_parsers/ident/xtandem_alanine.py

```diff
@@ -71,14 +71,7 @@ def __init__(self, *args, **kwargs):
         self.style = "xtandem_style_1"
         tree = ETree.parse(self.input_file)
         self.root = tree.getroot()
-        self.reference_dict.update(
-            {
-                "Raw data location": self.root.attrib["label"]
-                .split("models from ")[1]
-                .replace("'", ""),
-                "Search Engine": "xtandem_alanine",
-            }
-        )
+        self.reference_dict["Search Engine"] = "xtandem_alanine"
```

Comment:
How are parsers set up? for instance the engine wrappers are all the same, so e.g. for tandem_vengeance, would we have an additional parser inheriting from this one? otherwise we might need to add the option to fetch the engine name dynamically :)

Reply:
absolutely, added in new commit :)